src/xprof_vm_info.erl

```diff
@@ -13,38 +13,56 @@
 -spec get_available_funs(binary()) -> [MFA]
   when MFA :: list(). %% [module(), atom(), arity()]
 get_available_funs(Query) ->
-    case binary:split(Query, <<":">>) of
-        [ModBin, Rest] ->
-            case is_module(ModBin) of
-                {true, Mod} ->
-                    Funs = get_all_functions(Mod),
-                    %% match instead on "mod:fun/" if value seems to be a
-                    %% match-spec fun of the form "mod:fun(..."
-                    FunPrefix = re:replace(Rest, "\\(.*", "/", [{return, list}]),
-                    [[Mod, Fun, Arity]
-                     || {Fun, Arity} <- filter_funs(FunPrefix, Funs)];
-                false ->
-                    []
-            end;
-        [ModPrefix] ->
+    AllMods = get_modules(),
+    case find_mod(Query, AllMods) of
+        [{Mod, FunPrefix}] ->
+            Funs = get_all_functions(Mod),
             [[Mod, Fun, Arity]
-             || Mod <- filter_mods(binary_to_list(ModPrefix), get_modules()),
+             || {Fun, Arity} <- filter_funs(FunPrefix, Funs)];
+        [] ->
+            [[Mod, Fun, Arity]
+             || Mod <- filter_mods(Query, AllMods),
                 {Fun, Arity} <- get_global_functions(Mod)]
```

Comment:
BTW Do you know why do we call here `global_functions` instead of `get_all_functions` as in the previous clause? I found it a bit annoying when using xprof.